vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -1,33 +1,108 @@
 """Unit tests for vizro.models.Graph."""
+import json
+
+import plotly
 import pytest
 from pydantic import ValidationError
 
 import vizro.models as vm
 import vizro.plotly.express as px
+from vizro.actions import export_data
+from vizro.managers import data_manager
+from vizro.models._action._actions_chain import ActionsChain
+from vizro.models._components.graph import create_empty_fig
+
+
+class TestGraphInstantiation:
+    def test_create_graph_mandatory_only(self, standard_px_chart):
+        graph = vm.Graph(figure=standard_px_chart)
+
+        assert hasattr(graph, "id")
+        assert graph.type == "graph"
+        assert graph.figure == standard_px_chart._captured_callable
+        assert graph.actions == []
+
+    @pytest.mark.parametrize("id", ["id_1", "id_2"])
+    def test_create_graph_mandatory_and_optional(self, standard_px_chart, id):
+        graph = vm.Graph(
+            figure=standard_px_chart,
+            id=id,
+            actions=[
+                vm.Action(function=export_data()),
+            ],
+        )
+        actions_chain = graph.actions[0]
+
+        assert graph.id == id
+        assert graph.type == "graph"
+        assert graph.figure == standard_px_chart._captured_callable
+        assert isinstance(actions_chain, ActionsChain)
```

Comment:
The actions I would just leave as is, so no need to configure something here. The export_data example is actually quite confusing because we wouldn't configure that action on a Graph, but only on a Button or something similar. 

The action that would make sense with the Graph is the filter_interaction, but given that we thoroughly test them in the test_actions.py, I don't think you need to test these additionally